src/main/resources/assets/minecolonies/blockstates/blockhutsmeltery.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Take a look at the ForgeBlockSate Model. Would make reading tihs a lot bette.r

Reply:
okay done